overlord/snapstate/backend/link.go

```diff
@@ -135,15 +135,15 @@ func (b Backend) LinkSnap(info *snap.Info, dev boot.Device, prevDisabledSvcs []s
 	return reboot, nil
 }
 
-func (b Backend) StartServices(apps []*snap.AppInfo, meter progress.Meter, tm timings.Measurer) error {
-	return wrappers.StartServices(apps, meter, tm)
+func (b Backend) StartServices(apps []*snap.AppInfo, disabledSvcs []string, enableBeforeStart bool, meter progress.Meter, tm timings.Measurer) error {
+	return wrappers.StartServices(apps, disabledSvcs, enableBeforeStart, meter, tm)
```

Comment:
do we ever call this with enableBeforeStart = false ?

Reply:
No, but I didn't want to close these doors just yet, in particular for services.go; I don't have clarity yet since snap start has an optional --enable flag.